opensrp-unicef-tunisia/src/main/java/org/smartregister/uniceftunisia/fragment/AdvancedSearchFragment.java

```diff
@@ -50,45 +56,96 @@ protected BaseChildAdvancedSearchPresenter getPresenter() {
 
     @Override
     public boolean onBackPressed() {
-        ((BaseRegisterActivity) getActivity()).setSelectedBottomBarMenuItem(R.id.action_home);
+        if (getActivity() != null)
+            ((BaseRegisterActivity) getActivity()).setSelectedBottomBarMenuItem(R.id.action_home);
         return super.onBackPressed();
     }
 
+    @Override
+    protected void populateFormViews(View view) {
+        super.populateFormViews(view);
+        view.findViewById(R.id.child_birth_reg_number_layout).setVisibility(View.VISIBLE);
+        view.findViewById(R.id.child_unique_reg_layout).setVisibility(View.VISIBLE);
+        childRegistrationNumber = view.findViewById(R.id.child_birth_reg_number);
+        childUniqueGovtId = view.findViewById(R.id.child_reg);
+        enableCardSupport(view);
+        qrCodeButton.setVisibility(View.GONE);
+    }
+
+    private void enableCardSupport(View view) {
+        //TODO enable this when working on card support
+        View cardLayoutSupport = view.findViewById(R.id.card_support_layout);
+        cardLayoutSupport.setVisibility(View.VISIBLE);
+        cardLayoutSupport.setEnabled(false);
+
+        cardId = view.findViewById(org.smartregister.child.R.id.card_id);
+        cardId.addTextChangedListener(advancedSearchTextwatcher);
+        cardId.setEnabled(false);
+
+        Button scanCardQRCodeView = view.findViewById(R.id.scanCardButton);
+        scanCardQRCodeView.setText(R.string.scan_qr_code);
+        scanCardQRCodeView.setVisibility(View.VISIBLE);
+        scanCardQRCodeView.setEnabled(false);
+        scanCardQRCodeView.setOnClickListener(view1 -> {
+            if (getActivity() == null) {
+                return;
+            }
+            Utils.showShortToast(getActivity(), "Implement card support");
```

Comment:
will this be translated?

Reply:
It's just a placeholder will be removed completely.